Susi/Controllers/ChatViewController/ChatViewController.swift

```diff
@@ -73,11 +73,21 @@ class ChatViewController: UICollectionViewController {
     // send button
     lazy var sendButton: FABButton = {
         let button = FABButton()
-        button.setImage(ControllerConstants.Images.microphone, for: .normal)
-        button.addTarget(self, action: #selector(setTargetForSendButton), for: .touchUpInside)
-        button.accessibilityIdentifier = ControllerConstants.TestKeys.send
-        button.tintColor = UIColor.defaultColor()
-        button.backgroundColor = .clear
+        isSpeechToTextAvailable = UserDefaults.standard.bool(forKey: ControllerConstants.UserDefaultsKeys.speechToTextAvailable)
+        if isSpeechToTextAvailable && (speechRecognizer?.isAvailable)! {
```

Comment:
Instead use `if let _ = speechRecognizer.isAvailable, isSpeechToTextAvailable` just to be sure.